packages/autoloader/tests/php/test_plugins_handler.php

```diff
@@ -73,6 +73,15 @@ public function test_is_directory_plugin_single_file() {
 		$this->assertFalse( $this->plugins_handler->is_directory_plugin( 'test.php' ) );
 	}
 
+	/**
+	 * Tests is_directory_plugin() with an empty string.
+	 *
+	 * @covers Plugins_Handler::is_directory_plugin
+	 */
+	public function test_is_directory_plugin_single_file_with_slash() {
```

Comment:
```suggestion
	public function test_is_directory_plugin_single_file_with_empty_string() {
```

Duplicative function declaration.